src/Rector/Contrib/PHPUnit/SpecificMethodInternalTypeRector.php

```diff
@@ -0,0 +1,121 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Rector\Contrib\PHPUnit;
+
+use PhpParser\Node;
+use PhpParser\Node\Arg;
+use PhpParser\Node\Expr\FuncCall;
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\Node\Scalar\String_;
+use Rector\NodeAnalyzer\MethodCallAnalyzer;
+use Rector\NodeChanger\MethodNameChanger;
+use Rector\Rector\AbstractRector;
+
+/**
+ * Before:
+ * - $this->assertTrue(is_{internal_type}($anything), 'message');
+ * - $this->assertFalse(is_{internal_type}($anything), 'message');
+ *
+ * After:
+ * - $this->assertInternalType({internal_type}, $anything, 'message');
+ * - $this->assertNotInternalType({internal_type}, $anything, 'message');
+ */
+final class SpecificMethodInternalTypeRector extends AbstractRector
+{
+    /**
+     * @var string[]
+     */
+    private $oldMethodsToTypes = [
+        'is_array' => 'array',
+        'is_bool' => 'bool',
+        'is_callable' => 'callable',
+        'is_float' => 'float',
+        'is_integer' => 'integer',
+        'is_numeric' => 'numeric',
+        'is_object' => 'object',
+        'is_resource' => 'resource',
+        'is_scalar' => 'scalar',
+        'is_string' => 'string',
+    ];
+
+    /**
+     * @var string[]
+     */
+    private $renameMethodsMap = [
```

Comment:
Every `Rector` is getting this array... :thinking:

Reply:
Maybe time to create `AbstractChangeMethodNameAndMoveArgumentsUpRector`

or something like that :)